pkg/await/await.go

```diff
@@ -91,74 +57,21 @@ func Creation(
 		return nil, err
 	}
 
-	// Wait until create resolves as success or error.
+	// Wait until create resolves as success or error. Note that the conditional is set up to log only
+	// if we don't have an entry for the resource type; in the event that we do, but the await logic
+	// is blank, simply do nothing instead of logging an error.
```

Comment:
Minor - This and other comments say "instead of logging an error" - but this doesn't actually error in the `else` - just logs a info notice.